deltacat/compute/compactor_v2/utils/task_options.py

```diff
@@ -133,11 +137,15 @@ def hash_bucket_resource_options_provider(
     ray_custom_resources: Optional[Dict] = None,
     **kwargs,
 ) -> Dict:
+    debug_memory_params = {"hash_bucket_task_index": index}
```

Comment:
This is now redundant!

Reply:
I think it's ok to keep these, it makes the printed objects self-identifiable.

Reply:
I don't see them being read outside of the log. So, not sure why they have to self-identifiable. I have no strong bias either ways.